scripts/toaster_configuration

```diff
@@ -0,0 +1,153 @@
+#!/bin/sh
+#
+# Copyright 2015-2016 Mentor Graphics Corporation
+#
+# This file is licensed under the terms of the GNU General Public License
+# version 2.  This program  is licensed "as is" without any warranty of any
+# kind, whether express or implied.
+
+echo "Creating toaster configuration"
+
+BUILDDIR=$*
+
+BBLAYERS=
+configured_layers () {
+   tac "$BUILDDIR/conf/bblayers.conf" | \
+   sed -n -e '/^"/,/^BBLAYERS = /{ /^BBLAYERS =/d; /^"/d; p;}' | \
+   awk {'print $1'}
+}
+
+configured_layers | while read layer; do
+   if [ -d "$layer" ]; then
+       #Once testing is done, remove this if.
+       if [ -d "$layer/.git" ]; then
+            rm -fr "$layer/.git"
+       fi
+       cd "$layer"
+       if [ ! -d "$PWD/.git" ]; then
+            git init . > /dev/null 2>&1
+            sleep 1
+            git add . > /dev/null 2>&1
+            sleep 1
+            git commit -m "Initial commit" > /dev/null 2>&1
+            sleep 1
```

Comment:
What's with the pointless sleeps? git doesn't have timing issues, this is pointless.